dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -301,6 +305,8 @@ internal class DatadogRumMonitor(
     internal fun handleEvent(event: RumRawEvent) {
         if (event is RumRawEvent.AddError && event.isFatal) {
             rootScope.handleEvent(event, writer)
+        } else if (event is RumRawEvent.SynchronousEmptyEvent) {
```

Comment:
so we could do this or we could just make this an `EmptyEvent` and let it asynchronous. I am just afraid that this will turn back to us in some cases. I need your input here to see which way to go.